overlord/snapstate/snapstate_install_test.go

```diff
@@ -202,6 +207,24 @@ func (s *snapmgrTestSuite) TestInstallSnapdSnapType(c *C) {
 	c.Check(snapsup.Type, Equals, snap.TypeSnapd)
 }
 
+func (s *snapmgrTestSuite) TestInstallSnapdSnapTypeOnCore(c *C) {
```

Comment:
we will need either spread tests or managers_test.go tests as well for this

Reply:
Those are added now.